docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,44 @@
+= Enable snyk task for a product
+
+== Description:
+
+The sast-snyk-check task uses Snyk Code tool to perform Static Application Security Testing (SAST) for Snyk, a popular cloud-native application security platform.
+
+Snyk's SAST tool uses a combination of static analysis and machine learning techniques to scan an application's source code for potential security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.
+
+> NOTE: This task is executed only if the user provides a Snyk token stored in a secret in their namespace. The name of the secret then needs to be supplied in the `snyk-secret` pipeline parameter.
+
+== Steps to enable snyk:
+
+* Sign up/login at https://app.snyk.io/
+* Obtain Snyk token by navigating to the Account / General / Auth Token section
```

Comment:
--> . Get a Snyk token by selecting Account > General > Auth Token section.

I'm having trouble creating an acct/logging in so I can't see the click path myself, so to confirm: Those are the exact names of a user's click path? 

i also can't find the first one (Account) on the Snyk site.

Reply:
Could you try now with the updated text ?